src/docs/cookbook-events/like-feature/index.md

```diff
@@ -0,0 +1,527 @@
+  fetchLineItemsError: propTypes.error,
+```
+
+###### Step 10: Connect updateLikes to mapDispatchToProps function on ListingPage.js
+```diff
+const mapDispatchToProps = dispatch => ({
+  onManageDisableScrolling: (componentId, disableScrolling) =>
+    dispatch(manageDisableScrolling(componentId, disableScrolling)),
++  onUpdateLikes: (listingId, currentLikes) => dispatch(updateLikes(listingId, currentLikes)),
+  callSetInitialValues: (setInitialValues, values, saveToSessionStorage) =>
+    dispatch(setInitialValues(values, saveToSessionStorage)),
+  onFetchTransactionLineItems: (bookingData, listingId, isOwnListing) =>
+```
+
+
+## Update likes when clicking on the icon
+Currently, clicking on the icon will do nothing. We need to make some small tweaks for the button click to successfully update the user extended data. First, we'll need to pass props down through SectionHeading to SectionLikes. Then we'll add an onClick event handler to the React component in SectionLikes. 
+
+###### Step 1: Pass `SectionHeading` the correct props in `ListingPage.js`
+```diff
+<SectionHeading
+  priceTitle={priceTitle}
+  formattedPrice={formattedPrice}
+  richTitle={richTitle}
+  category={category}
+  hostLink={hostLink}
+  showContactUser={showContactUser}
+  onContactUser={this.onContactUser}
++ publicData={publicData} 
++ onUpdateLikes={onUpdateLikes} 
++ listingId={listingId.uuid} 
++ currentUser={currentUser} 
++ updateLikesInProgress={updateLikesInProgress}
+/>
+```
+
+###### Step 2: Define props in `SectionLikes.js`
+```diff
+const SectionLikes = props => {
+  const { 
+    publicData, 
++   onUpdateLikes, 
++   listingId, 
++   currentUser, 
++   updateLikesInProgress, 
+  } = props;
+```
+
+###### Step 3: Define currentLikes in `SectionLikes.js`
+```jsx
+const currentLikes = currentUser?.attributes?.profile?.privateData?.likedListings;
+```
+
+###### Step 4: Add an onClick event handler
+```diff
+<span className={css.heartIcon} 
++ onClick={() => {
++   if (!updateLikesInProgress && currentUser) {
++     onUpdateLikes(listingId, currentLikes);
++    }
++  }}>
+```
+
+Now, clicking on the icon will update either add or remove the listing to the user exteneded data with the key `likedListings`. However, as liked listings are only saved into user extended data at the moment, the number of likes still remains at zero. 
+
+## Listening to events
+Using events we can react to changes in users' extended data. To do this we'll listen to the /events/query endpoint, filter out relevant events and finally update listing extended data. We'll use the [`notify-new-listings.js`](https://github.com/sharetribe/flex-integration-api-examples/blob/master/scripts/notify-new-listings.js) script in the integration-api-examples as a basis for our new script. Make sure to follow the instructions at the [root of the repository](https://github.com/sharetribe/flex-integration-api-examples#getting-started) if you're unsure how to run the script locally.
```

Comment:
I'd suggest adding a link on the `/events/query endpoint` text pointing to the API ref